src/main/java/com/perennial/doctorappointmentbooking/service/AppointmentService.java

```diff
@@ -2,28 +2,19 @@
 import com.perennial.doctorappointmentbooking.entity.Appointment;
 import com.perennial.doctorappointmentbooking.helper.AppointmentHelper;
 import com.perennial.doctorappointmentbooking.repo.AppointmentRepo;
-import org.aspectj.weaver.patterns.ConcreteCflowPointcut;
-import org.hibernate.engine.spi.Status;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.Pageable;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Service;
-import org.springframework.util.ObjectUtils;
-import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.multipart.MultipartFile;
 import java.io.IOException;
-import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @Service
-public class AppointmentService {
+public class AppointmentService implements AppointmentInterface{
```

Comment:
Should be `AppointmentServiceImpl` implements `AppointmentService`. Do not use keywords like `Class` or `Interface` in the names